tests/test_perf_tracking.py

```diff
@@ -1974,6 +1954,16 @@ class TestPositionTracker(WithTradingEnvironment,
     ASSET_FINDER_EQUITY_SIDS = 1, 2
 
     @classmethod
+    def init_class_fixtures(cls):
+        super(TestPositionTracker, cls).init_class_fixtures()
+
+        cls.EQUITY1 = cls.asset_finder.retrieve_asset(1)
+        cls.EQUITY2 = cls.asset_finder.retrieve_asset(2)
+        cls.FUTURE3 = cls.asset_finder.retrieve_asset(3)
+        cls.FUTURE4 = cls.asset_finder.retrieve_asset(4)
+        cls.FUTURE5 = cls.asset_finder.retrieve_asset(1032201401)
```

Comment:
Was just porting the existing test over.  Don't know why it's such a high number.